core/src/main/java/org/wildfly/glow/error/Fix.java

```diff
@@ -72,6 +72,15 @@ public String getDescription() {
         return description;
     }
 
+    public boolean isEnv() {
+        return description.endsWith(" env");
```

Comment:
Just a random note. Should this have a leading space?

Reply:
Yeah, the way to findout that the fix is actually an env variable relies on this pattern.